mkosi

```diff
@@ -2819,34 +2829,38 @@ def build_stuff(args):
     # always the same
     args.machine_id = uuid.uuid4().hex
 
-    cache = setup_cache(args)
+    package_cache = setup_package_cache(args)
```

Comment:
This variable seems to be unused. It predates your change but since you are already modifying this line...